src/main.rs

```diff
@@ -3,10 +3,37 @@ mod mode;
 use mode::{cargo_test, execution};
 
 use anyhow::Result;
+use regex::Regex;
+use std::collections::VecDeque;
 use std::{env, path::PathBuf};
 
+// Lunatic versions under 0.13 implied run
+// This checks whether the 0.12 behaviour is wanted with a regex
+fn is_run_implied() -> bool {
+    if std::env::args().count() < 2 {
+        return false;
+    }
+
+    // lunatic <foo.wasm> -> Implied run
+    // lunatic run <foo.wasm> -> Explicit run
+    // lunatic fdskl <foo.wasm> -> Not implied run
+    let test_re = Regex::new(r"^(--bench|--dir|[^\s]+\.wasm)")
+        .expect("BUG: Regex error with lunatic::mode::execution::is_run_implied()");
```

Comment:
Didn't get to try this out yet, but would this work with .wasm files that have spaces in the name?

Reply:
Oh thanks - it does now :)

$ lunatic "foo bar.wasm"
```
Error: No such file or directory (os error 2)
```